schemas/1.0/maat_means.json

```diff
@@ -0,0 +1,352 @@
+              ]
+            },
+            "businesses":{
+              "type":"array",
+              "items":{
+                "type":"object",
+                "properties":{
+                  "name":{
+                    "type":"string"
+                  },
+                  "address":{
+                    "type":"array",
+                    "items":{
+                      "type":"object",
+                      "properties":{
+                        "lookup_id":{
+                          "anyOf":[
+                            {
+                              "type":"null"
+                            },
+                            {
+                              "type":"string"
+                            }
+                          ]
+                        },
+                        "address_line_one":{
+                          "type":"string"
+                        },
+                        "address_line_two":{
+                          "anyOf":[
+                            {
+                              "type":"null"
+                            },
+                            {
+                              "type":"string"
+                            }
+                          ]
+                        },
+                        "city":{
+                          "type":"string"
+                        },
+                        "country":{
+                          "type":"string"
+                        },
+                        "postcode":{
+                          "type":"string"
+                        }
+                      },
+                      "required":[
+                        "lookup_id",
+                        "address_line_one",
+                        "address_line_two",
+                        "city",
+                        "country",
+                        "postcode"
+                      ]
+                    }
+                  },
+                  "last_two_years_tax_liability":{
+                    "type":"integer"
+                  },
+                  "total_turnover":{
+                    "type":"integer"
+                  },
+                  "total_drawing":{
+                    "type":"integer"
+                  },
+                  "total_profit":{
+                    "type":"integer"
+                  },
+                  "director_salary":{
+                    "type":"integer"
+                  },
+                  "share":{
+                    "type":"number"
+                  }
+                },
+                "required":[
+                  "name",
+                  "address",
+                  "last_two_years_tax_liability",
+                  "total_turnover",
+                  "total_drawing",
+                  "total_profit",
+                  "director_salary",
+                  "share"
+                ]
+              }
+            }
+          },
+          "required":[
+            "paye"
```

Comment:
Why is paye required but not businesses?

Reply:
I am not sure, this is a subset of the overall means.json though and it is not a requirement in that schema either.

Reply:
Right, ok